packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1722,7 +1726,7 @@ export function catalogServiceBuilder(
       const recentDescriptorVersion = Math.max(...descriptorVersions);
 
       // eslint-disable-next-line @typescript-eslint/explicit-function-return-type
-      const event = () => {
+      const data = async () => {
```

Comment:
Could we name this with a verb explaining what the function does?
Something like 👇  

```suggestion
      const createActivationEvent = async () => {
```